app/javascript/src/services/button.js

```diff
@@ -2,7 +2,7 @@ const SERVER_URL = process.env.API_HOST_URL || "http://localhost:3000";
 
 const getButton = ({ buttonId }) => {
   return buttonAction({
-    actino: "GET",
+    action: "GET",
```

Comment:
nice pickup, presumably `actino` was set but ignored? and a GET would have been performed by default? or did this raise an error?

Reply:
No, it didn't raise an error, I dug into it when I was looking for why rails could not find the Button :id in the params, so started digging into the imports JS